data.js

```diff
@@ -0,0 +1 @@
+let weekdays=["Mon","Tue","Wed","Thu","Fri","Sat","Sun"];
```

Comment:
Don't you think it is bettter to use the ```const```  keyword instead of ```let```? and could you please export the array so that we can use it in other part of our code?
```suggestion
 const weekdays=["Mon","Tue","Wed","Thu","Fri","Sat","Sun"]
```